src/dags/basiskaart.py

```diff
@@ -178,11 +194,22 @@ def create_basiskaart_dag(is_first, table_name, select_statement):
         grant_db_permissions = PostgresPermissionsOperator(task_id="grants", dag_name=dag_id)
 
     # Flow
-    slack_at_start >> create_tables >> copy_data >> change_data_capture >> create_mviews >> provenance_translation >> clean_up >> trigger_next_dag >> grant_db_permissions
+    (
+        slack_at_start
+        >> create_tables
+        >> copy_data
+        >> change_data_capture
+        >> create_mviews
+        >> provenance_translation
+        >> clean_up
+        >> trigger_next_dag
+        >> grant_db_permissions
+    )
 
     dag.doc_md = """
     #### DAG summary
-    This DAG contains BGT (basisregistratie grootschalige topografie) and KBK10 (kleinschalige basiskaart 10) and KBK50 (kleinschalige basiskaart 50) data
+    This DAG contains BGT (basisregistratie grootschalige topografie) and KBK10 (kleinschalige
+    basiskaart 10) and KBK50 (kleinschalige basiskaart 50) data
```

Comment:
Very small nitpick. These are not really semantic linebreaks (term I love since you introduced it :-)

Reply:
Good point. Will fix. I was a little to quick to satisfy flake8